lib/AST/ASTPrinter.cpp

```diff
@@ -390,6 +393,7 @@ static bool escapeKeywordInContext(StringRef keyword, PrintNameContext context){
     return isKeyword && keyword != "Self";
 
   case PrintNameContext::TypeMember:
+  case PrintNameContext::ModuleQualifiedType:
```

Comment:
Is there a reason to add a new enum case instead of using (and possibly renaming) `PrintNameContext::TypeMember`? Do we apply different logic to them somewhere else?

Reply:
We don't, my initial reasoning is that we're being explicit here about module qualification vs. looking in a nested type scope, but if y'all think that's not an interesting distinction I'm happy to just reuse `TypeMember`.